tests/resources/testplans/valid/testplan6.yaml

```
@@ -0,0 +1,46 @@
+description: >
+    A valid example testplan for demonstrating the usage of OpenTSDB
+    verification plugin
+verification:
+    -   states:
+          - STEADY
+        type: tsdb
+        config:
+            url: https://tsdb.resilience.yahoo.com/metrics
+            method: POST
+            query:
+                # Refer to OpenTSDB API to construct a query
+                start: '30m-ago'
+                end: 'now'
+                queries:
+                    -   aggregator: zimsum
+                        metric: system.cpu.used
+                        downsample: 15m-avg-nan
+                        filters:
+                            -   type: literal_or
+                                tagk: host
+                                filter: mockhost.resilience.cloud
```

```suggestion
                                filter: mockhost.resilience.yahoo.cloud
```
as TLD in the commit resolves to valid external dns entry

---

Addressed